README.md

```diff
@@ -115,6 +117,10 @@ const method = new Method({
 });
 ```
 
+#### Skipping argument validation
+
+If your method does not need argument validation, perhaps because it does not take any arguments, you can use `validate: true` to skip argument validation.
```

Comment:
Makes sense. I had trouble deciding whether true or false made more sense, so I think that is an argument in favor of null.